phoenix-core/src/test/java/org/apache/phoenix/execute/MutationStateTest.java

```diff
@@ -175,4 +185,43 @@ public void testGetMutationBatchList() {
 
     }
 
+    @Test(expected = SQLException.class)
+    public void testPendingMutationsOnDDL() throws Exception {
+        Connection conn = null;
+        PhoenixConnection spy = null;
+        try {
+            Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+            props.setProperty(QueryServices.PENDING_MUTATIONS_DDL_THROW_ATTRIB, "true");
+            conn = DriverManager.getConnection(getUrl(), props);
```

Comment:
Surround connection with try-with-resources for auto-closing.

Reply:
done